pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -145,6 +145,16 @@ type NetworkPolicyController struct {
 	// heartbeatCh is an internal channel for testing. It's used to know whether all tasks have been
 	// processed, and to count executions of each function.
 	heartbeatCh chan heartbeat
+
+	// initPodSet caches UIDs of the Pods that exist before starting computation.
+	// We don't need to handle their ADD events as all AddressGroups and AppliedToGroups
+	// will be computed once at the beginning and they have counted these Pods in.
```

Comment:
instead of "all AddressGroups and AppliedToGroups will be computed once at the beginning and they have counted these Pods in.", do you think it would be more accurate to say "all AddressGroups and AppliedToGroups will be computed when the ADD events for NetworkPolicies are processed and they will account for these Pods"?